### PR TITLE
WIP - Python3 import cleanup

### PR DESCRIPTION
**Pull request by tfmorris**
Closes #

### CODE REVIEW

**Comment by tfmorris** [comment]
@cclauss Is there anything here which isn't included in #2888? If so, let's just cherrypick it onto that PR. It doesn't make sense to duplicate work. 

Perhaps I'll temporarily repurpose this branch to integrate all the pending fixes to see what else is needed, but eventually I think it'll just get closed.

**Comment by cclauss** [comment]
I will scan now...

I like your idea of an aggregate draft that incorporates:
#2816 + #2866 + #2888 + #2890 + #2892 + #2893

**Comment by cclauss** [comment]
#2416 + #2888 are a superset of this PR.  You can proceed to make this the aggregate draft.

**Comment by cclauss** [comment]
Needs https://github.com/internetarchive/openlibrary/pull/2888/commits/ccdd0e94df29e29f417325deb8aabf7d8a0061c4 and #2890

File: openlibrary/coverstore/utils.py [hunk 1]
```diff
@@ -36,12 +31,14 @@ def safeint(value, default=None):
     """
     try:
         return int(value)
-    except:
+    except ValueError:
```
Comment (cclauss) [inline openlibrary/coverstore/utils.py]: int('a') --> ValueError
int([]) --> TypeError

**Comment by cclauss** [comment]
@tfmorris Would be willing to rebase this?  It would be good to land the remaining changes.

**Comment by tfmorris** [comment]
There are too many commits on this branch to attempt a rebase, but I merged the current master and, presuming I didn't screw up the merge, it looks like pretty much everything has been covered on the `import` front.

Since #2897 built on this work to start addressing test failures, I'll probably just switch to that and update it to reflect the current master. I'm not sure if I'll cherrypick the last dozen commits onto a new branch or just merge the master branch into it.

@cclauss Either way, I'll close this unless you see specific items to pull from the remaining diffs.

**Comment by cclauss** [comment]
I think we can close.  Thanks.

**Pull request closed without merging**